clang/lib/CConv/ProgramInfo.cpp

```diff
@@ -188,7 +178,8 @@ void ProgramInfo::print_stats(const std::set<std::string> &F, raw_ostream &O,
       ConstraintVariable *C = I.second;
       if (C->isForValidDecl()) {
         InSrcCVars.insert(C);
-        CAtoms FoundVars = getVarsFromConstraint(C);
+        CAtoms FoundVars;
+        getVarsFromConstraint(C, FoundVars);
```

Comment:
This is a surprising refactoring; avoids a copy, maybe?

Reply:
Since `getVarFromConstraints` is defined recursively, there would be a copy at each level of the call stack.